README.md

```diff
@@ -252,26 +252,28 @@ There are two deprecated repositories containing [FAQ](https://github.com/ipfs/f
 
 #### IPFS GUIs
 
-- [ipfs-companion](https://github.com/ipfs-shipyard/ipfs-companion) - The web browser extension
-- [ipfs-desktop](https://github.com/ipfs-shipyard/ipfs-desktop) - A menubar/tray desktop app
-- [ipfs-webui](https://github.com/ipfs/webui) - The IPFS WebUI app
-- [i18n](https://github.com/ipfs/i18n) - The IPFS Translation Project: crowdsourcing translations of IPFS GUIs and websites
-- [pm-ipfs-gui](https://github.com/ipfs-shipyard/pm-ipfs-gui) - Coordinating development and maintenance of GUI apps
+- [ipfs-companion](https://github.com/ipfs-shipyard/ipfs-companion) - The web browser extension.
+- [ipfs-desktop](https://github.com/ipfs-shipyard/ipfs-desktop) - A menubar/tray desktop app.
+- [ipfs-webui](https://github.com/ipfs/webui) - The IPFS WebUI app.
+- [i18n](https://github.com/ipfs/i18n) - The IPFS Translation Project: crowdsourcing translations of IPFS GUIs and websites.
+- [pm-ipfs-gui](https://github.com/ipfs-shipyard/pm-ipfs-gui) - Coordinating development and maintenance of GUI apps.
 
 #### Apps on IPFS
 
 You can find an ever growing list at [Awesome IPFS](https://awesome.ipfs.io/categories/apps/)
 
 #### Community Infrastructure
 
-- [blog](https://github.com/ipfs/blog) - The IPFS Blog
+- [blog](https://github.com/ipfs/blog) - The IPFS Blog.
 - [community forum](https://discuss.ipfs.io)
 - [distributions](https://github.com/ipfs/distributions) - Scripts to build the /install html page.
-- [newsletter](https://github.com/ipfs/newsletter) - Prepare and store IPFS Newsletter roundups
+- [newsletter](https://github.com/ipfs/newsletter) - Prepare and store IPFS Newsletter roundups.
 - [infrastructure](https://github.com/ipfs/infrastructure) - Tools and systems for the community.
-- [ops-requests](https://github.com/ipfs/ops-requests) - Requests about infrastructure operations
-- [project-repos](https://github.com/ipfs/project-repos) CI status and other health metrics
+- [ops-requests](https://github.com/ipfs/ops-requests) - Requests about infrastructure operations.
+- [project-repos](https://github.com/ipfs/project-repos) CI status and other health metrics.
 - [website](https://github.com/ipfs/website) - The source to the IPFS community website at http://ipfs.io.
+- [IPFS group on Linkedin](https://www.linkedin.com/groups/13635533/) - Open group to share and learn about IPFS on Linkedin. 
```

Comment:
We don't link to Reddit ([/r/ipfs/](https://www.reddit.com/r/ipfs/)), so makes little sense to add Linked-in. 
AFAIK as a rule of thumb, we don't link  to communities at third party websites outside of our official infrastructure and control as it gets nasty really quick: what are criteria for "endorsing" a community? how do we control abuse and moderation? not to mention confusing and splitting community into tiny silos.

I vote for removing this line:

```suggestion

```

Reply:
That sounds totally reasonable. Each newsletter is constructed here: https://github.com/ipfs/newsletter/pulls.

Reply:
Linkedin group , wait there, i m coming~~~